app/controllers/MathSwipeController.coffee

```diff
@@ -33,26 +34,25 @@ class MathSwipeController
       TrackingService.desktopView()
       @cursorToPointer()
     ShareGameService.setMessage()
-    @initialize window.location.hash
+    @initialize window.location.hash, 3
```

Comment:
lets change this to ``` @initialize window.location.hash, length ``` where length is an updated constant so it can work better with the leveling system